packages/backend/src/modules/user/user.service.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable class-methods-use-this */
+import { CreateUserDto, UpdateUserDto } from 'shared'
+import { User } from './user'
+import userRepository, { UserRepository } from './user.repository'
+
+export class UserService {
```

Comment:
Users are stored not only in UserModel but also in Dashboard. We use duplications to avoid unnecessary calls.
Check dashboard model

```
{
    _id: ObjectId('622e5dd9c3cba876b8800f93'),
    title: 'Tablica 1',
    description: 'Opis tablicy pierwszej',
    createdAt: 'Fri May 06 2022 20:16:40 GMT+0000 (UTC)',
    status: 'PUBLIC',
    users: [
        {
            _id: ObjectId('622e5dffa63863cfc5f812f5'),
            username: 'Marek Kowalski',
            avatarId: 345
        }
    ],
    columns: [
        {
            _id: ObjectId('622e5e084eb135ee631dcd8e'),
            title: 'Kolumna Pierwsza',
            tasks: [
                {
                    _id: ObjectId('622e5e109a5d1fb63a360fb0'),
                    title: 'Tytuł 1',
                    imageCoverId: 'photo-1646388023469-18941b7e3b6f',
                    comments: [
                        {
                            _id: ObjectId('622e5e179c945199e651891a')
                        }
                    ],
                    attachments: [
                        {
                            _id: ObjectId('622e5e1c15be96a29a180abe')
                        }
                    ],
                    users: [
                        {
                            _id: ObjectId('622e5e2094298d670e6129ee')
                        }
                    ]
                }
            ]
        }
    ]
}
```

Reply:
Yeah, I (wrongly) assumed that you meant I should change the User model to only store dashboard IDs in the `memberOfDashboards` field https://github.com/kacper-cyra/camp-trello-clone/pull/112#discussion_r873225690

Reply:
However I still think it would be better to use 
```javascript
dashboards: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Dashboard' }]
```
and then call `populate('dashboard')` in the mongoose query